data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -1,5 +1,28 @@
 ---
 queries:
+  - action: ENUM_ADCS_CAS
+    description: 'Enumerate ADCS certificate authorities.'
+    base_dn_prefix: 'CN=Enrollment Services,CN=Public Key Services,CN=Services,CN=Configuration'
+    filter: (&(objectClass=pKIEnrollmentService))
```

Comment:
Your not doing multiple elements here so not sure why this is using an AND filter?

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/16938/commits/109065e7c7321dd7b624430f562814398decf77d